src/sagemaker/image_uri_config/huggingface-llm-neuronx.json

```diff
@@ -4,7 +4,7 @@
             "inf2"
         ],
         "version_aliases": {
-            "0.0": "0.0.16"
+            "0.0": "0.0.18"
```

Comment:
I would recommend we not change this for now. There are certain backward-compatibility issues that not are not being accounted for in the traditional major/minor/patch versioning possibly due to this currently still being in "development" phase. This is why I did not change it to 17 (intentional, not something that was missed).